### PR TITLE
fix: fixed misaligned navbar icons

### PR DESCRIPTION
**Pull request by wane-nwani**
## Purpose
This pull request will fix this [issue](https://github.com/ambianic/ambianic-ui/issues/676)

## Approach
This pull request adds a `v-spacer` element to ensure enough space between the menu icon and the three navbar icons.

## Merge Checklist
- [x]  The code change is tested and works locally.
- [x]  The user and dev documentation is up to date.
- [x]  There is no commented out code in this PR.
- [x]  No lint errors (use flake8)
- [x]  100% test coverage

### CODE REVIEW

**Comment by ivelin** [comment]
@vickywane [deploy preview](https://deploy-preview-677--ambianic-pwa-dist.netlify.app/) shows a number of errors in the dev console that also trigger failures in the Lighthouse check. The first error originates in the onboarding code. See screenshot.

Screen Shot 2021-07-21 at 5 21 20 PM

Related [lighthouse check errors](https://github.com/ambianic/ambianic-ui/pull/677/checks?check_run_id=3124390274#step:4:155):

Screen Shot 2021-07-21 at 5 25 15 PM

There are also[ lighthouse report errors ](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1626874470347-48993.report.html) related to loading custom materialdesign icons:

It is strange to see that there are calls to Segment. We are not using this proprietary analytics site. Whichever library is using it, needs to be removed as this is a privacy issue.


Screen Shot 2021-07-21 at 5 24 03 PM

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
     fluid
   >
     <v-app-bar
-      app
+      app1
```
Comment (ivelin) [inline src/components/NavBar.vue:8]: @vickywane what is the purpose of renaming app to app1?

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -18,6 +18,7 @@
         />
         <span class="hidden-sm-and-down">Ambianic</span>
       </v-toolbar-title>
+      <v-spacer />
```
Comment (ivelin) [inline src/components/NavBar.vue:21]: @vickywane v-spacer appears to be an adequate fix to the issue of navbar icon alignment. However I now observe an overlap of the timeline pane over the navbar on bigger screens. See screenshot. This timeline issue is not present on small screens.

Screen Shot 2021-07-21 at 5 33 16 PM

Screen Shot 2021-07-21 at 5 37 40 PM

Comment (ivelin) [reply src/components/NavBar.vue:21]: BTW, this timeline pane overlap issue is not present on big screens nor small screens with the current production deployment. See screenshots below.

Screen Shot 2021-07-21 at 5 39 26 PM

Screen Shot 2021-07-21 at 5 40 16 PM

**Review by ivelin — CHANGES_REQUESTED** [review]

File: src/views/Timeline.vue [hunk 3]
```diff
@@ -42,16 +42,16 @@
               lazy-src="/img/lazy-load-bg.gif"
             >
               <template #placeholder>
-                    <v-row
-                    class="fill-height ma-0"
```
[the remark this answers is not in the document]
Comment (vickywane) [reply src/views/Timeline.vue:46]: The code here is the same. The change was caused due to linting the codebase using yarn lint before pushing.

Looking at this from another angle, it is related to the problem of code contributions not being linted before a push is made. If the merged code was linted, this change would have not been made.

I have [created this GitHub issue](https://github.com/ambianic/ambianic-ui/issues/678) to track and implement automated linting.

File: src/components/NavBar.vue (hunk 1 above)
Comment (vickywane) [reply src/components/NavBar.vue:8]: Was from a previous change. I have restored it back to `app`

**Comment by vickywane** [comment]
> deploy preview shows a number of errors in the dev console that also trigger failures in the Lighthouse check. The first error originates in the onboarding code. See screenshot.

I debugged these errors and they traced back to the `get` method within the `PeerFetch` in the `peer-fetch.js` file. The asynchronous `_receiveResponse` method was being invoked without a `try/catch`. The `_receiveResponse` will intentionally throw a timeout error ( which should be caught in the `try/catch` ) when no Peer connection is established. 

Also, it had nothing to do with the Onboarding screen. The `INITIALIZE_PNP` action is dispatched to instantiate the PeerFetch class immediately the app is loaded.

**Comment by vickywane** [comment]
The white overlap in the NavBar has been fixed and can be tested from [this deploy preview ](https://deploy-preview-677--ambianic-pwa-dist.netlify.app/) here.

**Comment by vickywane** [comment]
It's quite difficult for me to find the library using Segment. We don't have any use for those API within the application. 

Could it be from Netlify's deploy preview ( probably injected to track any errors while generating a Netlify preview) ?

**Comment by ivelin** [comment]
> > deploy preview shows a number of errors in the dev console that also trigger failures in the Lighthouse check. The first error originates in the onboarding code. See screenshot.
> 
> I debugged these errors and they traced back to the `get` method within the `PeerFetch` in the `peer-fetch.js` file. The asynchronous `_receiveResponse` method was being invoked without a `try/catch`. The `_receiveResponse` will intentionally throw a timeout error ( which should be caught in the `try/catch` ) when no Peer connection is established.
> 
> Also, it had nothing to do with the Onboarding screen. The `INITIALIZE_PNP` action is dispatched to instantiate the PeerFetch class immediately the app is loaded.
> 
> I believe this should fix the error as i can't find the errors in console for the latest deploy preview.

This analysis makes sense. How about the code that depends on this peer get() method. Would it be able to gracefully recover from a method that returns nothing in its catch{} block?

**Comment by ivelin** [comment]
> It's quite difficult for me to find the library using Segment. We don't have any use for those API within the application.
> 
> Could it be from Netlify's deploy preview ( probably injected to track any errors while generating a Netlify preview) ?

That is possible. Let's keep an eye on it and look for ways to turn off. User tracking without explicit authorization is illegal.

**Review by ivelin — APPROVED** [review]
@vickywane I will approve this PR with a few comments to follow up on:
1. User tracking via Segment.io
2. Handling of peer.get() that returns no result. 

Please address these as separate issues.

**Comment by vickywane** [comment]
> I will approve this PR with a few comments to follow up on:

    User tracking via Segment.io
    Handling of peer.get() that returns no result.

I will take note of these

**Pull request merged**